### PR TITLE
#3156; disable config of auto selection for autocomplete

### PR DESCRIPTION
**Pull request by OBreidenbach**

### CODE REVIEW

File: src/main/resources/META-INF/resources/primefaces/autocomplete/autocomplete.js [hunk 1]
```diff
@@ -809,9 +810,9 @@ PrimeFaces.widget.AutoComplete = PrimeFaces.widget.BaseWidget.extend({
 
             for(var i = 0; i < $this.currentItems.length; i++) {
                 var stripedItem = $this.currentItems[i];
```
Comment (melloware) [inline src/main/resources/META-INF/resources/primefaces/autocomplete/autocomplete.js:812]: I know you didn't make this variable but shouldn't "stripedItem" be "strippedItem" ?

**Review by melloware — APPROVED** [review]

File: src/main/resources/META-INF/resources/primefaces/autocomplete/autocomplete.js (hunk 1 above)
Comment (OBreidenbach) [reply src/main/resources/META-INF/resources/primefaces/autocomplete/autocomplete.js:812]: I think you are right, it should be "strippedItem".

Comment (Rapster) [reply src/main/resources/META-INF/resources/primefaces/autocomplete/autocomplete.js:812]: I did! My bad 😅

**Comment by mertsincan** [comment]
Thanks for the PR ;)

**Pull request merged**